### ISSUE TITLE
Missing columns titles for List view on Recent tab

### ISSUE THREAD

**Issue by ghost** [xwiki-labs/cryptpad#1233]
### Description

File/Folder name, # of folders, # of files, Type, Last access & Creation are missing as column titles.

<img width="935" alt="Screenshot 2023-09-06 at 08 52 58" src="https://github.com/cryptpad/cryptpad/assets/41020854/eddbf933-550a-426d-904b-1889309164c9">

On other views, they are properly shown:
<img width="1025" alt="Screenshot 2023-09-06 at 08 56 24" src="https://github.com/cryptpad/cryptpad/assets/41020854/91b53f95-20b0-46f9-87a9-04d0df9fbf6b">

### Steps to reproduce

1. Login into your account
2. Go to your drive
3. Set "List view" instead of the default "Grid view"
4. Select "Recent" in the left sidebar
5. Witness the missing column titles

### Expected behavior

Column titles should be shown, otherwise there is no way to make sense of the dates.

### What opreating system are you using?

Linux/BSD/UNIX, Apple macOS

### What browsers are you seeing the problem on?

Mozilla Firefox, Google Chrome, Apple Safari

### Extensions

uBlock Origin, Wappalyzer & Passbolt

### Version

5.4.1
